suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -115,17 +116,29 @@ const connectDevice = (draft: State, device: Device) => {
     // fill draft with not affected devices
     otherDevices.forEach(d => draft.devices.push(d));
 
-    // prepare new device
+    const deviceInstance = features.passphrase_protection
+        ? deviceUtils.getNewInstanceNumber(draft.devices, device) || 1
+        : undefined;
+
+    const isMobile = getEnvironment() === 'mobile';
+
+    let useEmptyPassphrase: boolean;
```

Comment:
nitpick, personal taste - I don't like variables in code if it's not performance optimalization or something like that, I found constants easier to think about and easier to read 😉  So I would prefer to write it in a function. But feel free to ignore...